helpline-dialer.php

```diff
@@ -14,7 +14,12 @@
     if ($call_status != "completed") { ?>
         <Dial method="GET"
           timeout="<?php echo $call_timeout?>"
-          callerId="<?php echo isset($_REQUEST["Called"]) ? $_REQUEST["Called"] : "0000000000" ?>"
+          callerId="<?php
+          if (isset($GLOBALS['outbound_callerid']) && $GLOBALS['outbound_callerid']) { 
+            echo isset($GLOBALS["outbound_callerid"]) ? $GLOBALS["outbound_callerid"] : "0000000000"; 
+          } else { 
+              echo isset($_REQUEST["Called"]) ? $_REQUEST["Called"] : "0000000000";  
+          } ?>"
```

Comment:
I meant to have an if block with variable assignment. this would be after line 14 before line 15.

Make a new variable called $caller_id which has the result of the if block.

then on what is currently line 17 you'd have `<?php echo $caller_id ?>`